src/components/ChecSlideoutPanel/Panel.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <transition
+    name="slide"
+    mode="out-in"
+  >
+    <div
+      class="slideout-panel__element"
+      :class="[classNames, `sm:w-${size}`]"
+    >
+      <div class="slideout-panel__header">
+        <div class="slideout-panel__header-inner">
+          <component :is="titleTag" v-if="title" class="slideout-panel__title">
+            {{ title }}
+          </component>
+          <ChecButton
+            class="slide-panel__close"
+            color="secondary"
+            variant="small"
+            @click="emitClose"
```

Comment:
I'd like to be able to press the Escape key and have it close the panel, could you add that too please?